libpromises/evalfunction.c

```diff
@@ -923,6 +923,148 @@ static FnCallResult FnCallGetUsers(ARG_UNUSED EvalContext *ctx, ARG_UNUSED const
 
 /*********************************************************************/
 
+
+#if defined(HAVE_GETPWENT) && !defined(__ANDROID__)
+
+static FnCallResult FnCallFindLocalUsers(EvalContext *ctx, ARG_UNUSED const Policy *policy, const FnCall *fp, const Rlist *finalargs)
+{
+    bool allocated = false;
+    JsonElement *json = VarNameOrInlineToJson(ctx, fp, finalargs, false, &allocated);
+    
+    // we failed to produce a valid JsonElement, so give up
+    if (json == NULL)
+    {
+        Log(LOG_LEVEL_VERBOSE,
+            "Cannot produce valid JSON from the argument '%s' of the function '%s'",
```

Comment:
Maybe you can come up with a message that helps the users understand what they did wrong. The user does not need to know that the arguments are converted into a JSON data structure.

Although, messages like these are nice to help other developers figure out what is wrong. But such messages belong in `LOG_LEVEL_DEBUG`.